### DISCUSSION TITLE
Question: Does OrchardCore with RCL support changing wwwroot path?

### DISCUSSION THREAD

**Discussion by iyilm4z** [OrchardCMS/OrchardCore#10550] [Q&A]
Hi,
I would like to store **wwwroot** folder inside of **Web** folder. I tried many possible solutions on the web but i couldn't make it working with plain AspNetCore. Is it possible with OrchardCore?

![image](https://user-images.githubusercontent.com/9105939/138604639-de711995-7970-4d0e-808f-ce0dcabe4020.png)

**Comment by ns8482e** [comment]
It's not orchard related question - may be you can achieve using `UseWebRoot` method of Host Builder

https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.hosting.hostingabstractionswebhostbuilderextensions.usewebroot?view=aspnetcore-5.0

**Reply by iyilm4z** [reply]
Ok i'll try.
